### PR TITLE
Remove package-lock, we use yarn

### PR DESCRIPTION
**Pull request by zencephalon**
**What changed? Why?**

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

**Comment by lukasrosario** [comment]
maybe add to gitingnore just in case it happens again

**Review by lukasrosario — APPROVED** [review]

**Comment by zencephalon** [comment]
@lukasrosario yeah that's a great idea, could U re-review? Thanks

File: .gitignore [hunk 1]
```diff
@@ -36,18 +36,6 @@ local.properties
 *.iml
 *.hprof
 
-# Nx/Builds
-.docusaurus
```
Comment (lukasrosario) [inline .gitignore:40]: agree that we probably dont need all of these now but i dont think we want to remove all of them. eg building docs still produces a `.docusaurus` directory. we should probably revisit to see what exactly can be removed & just add the package lock for now

Comment (zencephalon) [reply .gitignore]: Yeah, very good, added those back.

**Review by wbnns — APPROVED** [review]

**Pull request merged**